doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -79,8 +98,16 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 |`ra2.k8s.003`|Highly available control plane|An implementation must consist of at least one master node per availability zone or fault domain to ensure the high availability and resilience of the Kubernetes control plane services|[req.gen.rsl.02 req.gen.avl.01](./chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.k8s.004`|Highly available worker nodes|An implementation must consist of at least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes|[req.gen.rsl.01 req.gen.avl.01 req.kcm.gen.02](./chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.k8s.005`|Kubernetes API Version|In alignment with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), an implementation must use one of three latest minor versions (`n-2`). e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15.|TBC|
-|`ra2.k8s.006`||||
-|`ra2.k8s.007`||||
+|`ra2.k8s.006`|NUMA Support|The `TopologyManager` and `CPUManager` feature gates must be enabled and configured on the kubelet (note, TopologyManager is enabled by default in Kubernetes v1.18 and later, with CPUManager enabled by default in Kubernetes v1.10 and later). `--feature-gates="...,TopologyManager=true,CPUManager=true" --topology-manager-policy=single-numa-node --cpu-manager-policy=static`|[e.cap.007](./chapter02.md#221-cloud-infrastructure-software-profile-capabilities) [infra.com.cfg.002](./chapter02.md#223-cloud-infrastructure-software-profile-requirements) [infra.hw.cpu.cfg.004](./chapter02.md#224-cloud-infrastructure-hardware-profile-requirements)|
+|`ra2.k8s.007`|DevicePlugins Feature Gate|In order to support the various acceleration devices required, the DevicePlugins feature gate must be enabled (note, this is enabled by default in Kubernetes v1.10 or later). `--feature-gates="...,DevicePlugins=true,..."`|Various, e.g. [e.cap.013, e.cap.014](./chapters/chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.k8s.008`|System Resource Reservations|To avoid resource starvation issues on nodes, reserve compute resources for system daemons and Kubernetes system daemons such as kubelet, container runtime, etc. (requires Kubernetes version 1.17 or later). Use the following kubelet flags: `--reserved-cpus=[0-3]`|NB: There doesn't appear to be a requirement that this is for?  Do we need to be specific about the values used?  The guidelines [here](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#general-guidelines) suggest a more nuanced approach is recommended?|
```

Comment:
number of CPUs to be reserved always depends on the stuff running on the specific infrastructure. even within the same product differetn configurations can be imagined, requiring different numbers (with / without Ceph etc.)

Reply:
Yeah I'm a little nervous about including this tbh (partly as you've described, partly because there is nothing in chapter02.md suggesting it's a requirement, partly that it feels a little out of scope of what we're trying to focus on in CNTT).

Reply:
yeah I would suggest to keep it on the "configure enough CPU cores for your host platform via --reserved-cpus"
additional note: you separately mention static policym ust be enabled so functionally speaking --reserved-cpus will work, but it worth noting --reserved-cpus **only works** with static CPU policy